minimal-decision-environment.yml

```diff
@@ -19,7 +19,7 @@ dependencies:
     - dpath
     - ansible-rulebook
   ansible_core:
-    package_pip: ansible-core==2.14.4
+    package_pip: ansible-core==2.16
```

Comment:
No idea if this file is still used.

Reply:
I think it is just as reference for DE building.